src/tactic/ui/tools/trigger_wdg.py

```diff
@@ -1485,6 +1484,7 @@ def get_display(my):
             outputs_div.add(output_div)
             checkbox = CheckboxWdg("output")
             checkbox.set_option("value", output)
+            checkbox.add_attr("spt_is_multiple", "true")
```

Comment:
This is necessary because the function spt.api.utility.get_input_values requires the spt_multiple class to allow selection of multiple input checkboxes.